service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DocumentReferenceToNarrativeStatementMapperTest.java

```diff
@@ -213,7 +265,7 @@ public void When_MappingParsedDocumentReferenceJsonWithContentAndNoAttachment_Ex
     }
 
     @Test
-    public void When_MappingParsedDocumentReferenceJsonWithNoAttachmentContentType_Expect_MapperException() {
+    void When_MappingParsedDocumentReferenceJsonWithNoAttachmentContentType_Expect_MapperException() {
```

Comment:
I think `json` isn't required here, as it's the GP2GP adaptor I think this is OK.

```suggestion
    void When_MappingParsedDocumentReferenceWithNoAttachmentContentType_Expect_MapperException() {
```